### ISSUE TITLE
[ff89c9]Update the rule and examples to comply with ARIA 1.3

### ISSUE THREAD

**Issue by shunguoy** [auto-wcag/auto-wcag#2439]
Aria 1.3 replaced "ARIA required context role" with Required Accessibility Parent Role](https://w3c.github.io/aria/#scope). The suggested changes are:

- Update the rule name: from "ARIA required context role" with "ARIA required accessibility parent role" ??
- Update terms, links
- Optionally add a new passed example based on 1.3
- Optionally ass a new failed example based on 1.3
- Review Failed Example 3. based on [Relationships in the Accessibility Tree ](https://w3c.github.io/aria/#tree_relationships) in ARIA 1.3, this example is in fact a passed example to me:

```
<div role="list">
	<div aria-live="polite">
		<div role="listitem">List item 1</div>
		<div role="listitem">List item 2</div>
	</div>
</div>
```

 The `<div aria-live="polite">` has a 'generic' role, therefore, the `listitem` elements's parent is the list element even though the existence of `<div aria-live="polite">` in the hierarchy based on [Relationships in the Accessibility Tree ](https://w3c.github.io/aria/#tree_relationships) in ARIA 1.3:     
    "the accessibility parent is one of the following:
       ....
    The DOM ancestor of the [element](https://dom.spec.whatwg.org/#concept-element) with only elements of role [generic](https://w3c.github.io/aria/#generic) or [none](https://w3c.github.io/aria/#none) intervening.
   "

- Any other changes needed.

 @dan-tripp-siteimprove could you please work on this if you get some time? 
@WilcoFiers

**Comment by dan-tripp-siteimprove** [comment]
Will do, in the next few days I hope.  I'm still catching up from my vacation.
